mlflow/go/tools/generate/endpoints.go

```diff
@@ -21,7 +21,7 @@ var ServiceInfoMap = map[string]ServiceGenerationInfo{
 }
 
 var ImplementedEndpoints = []string{
-	// "MlflowService_getExperimentByName",
+	"MlflowService_getExperimentByName",
```

Comment:
Do you think we could also rename the services here? (e.g. `TrackingService_getExperimentByName`)

Reply:
No, these are names used from the proto files.